### ISSUE TITLE
Next release

### ISSUE THREAD

**Issue by joachimwolff** [deeptools/pyGenomeTracks#29]
Hi Fidel,

I updating the hicPlotTads Galaxy wrapper to be pyGenomeTracks wrapper and I have seen that since the last 2.0 release you added 50+ commits to master. I would like to include the latest changes to a new pyGenomeTracks wrapper.  Do you plan to release a new version soon? 

Best,

Joachim

**Comment by vivekbhr** [comment]
@joachimwolff we planned the next release after #6 is working. currently, we lack the ability to add own color list for the epilogos.

**Comment by crazyhottommy** [comment]
Thanks for taking care of https://github.com/deeptools/pyGenomeTracks/issues/6  waiting for this for so long...

**Comment by joachimwolff** [comment]
Any progress on this? I replaced today the dependency to HiCExplorer with the new HiCMatrix (branch develop). With this we can integrate pyGenomeTracks in HiCExplorer, but I need a new release for this.

**Comment by vivekbhr** [comment]
@joachimwolff you mean a pyGT release that switches to the import of HiCMatrix ? I would have to test the HiCMatrix import first

**Comment by joachimwolff** [comment]
1. Sure we need to test it.
2. It was written you wait until #6 is implemented for the next release. I wondered if this is already achieved or not.

**Comment by vivekbhr** [comment]
Yes #6 is implemented (not merged to master yet)

**Comment by joachimwolff** [comment]
Ok, great :+1: 

I will create a new PR for the changes as soon as I have the conda package.
